src/index.js

```diff
@@ -88,7 +88,7 @@ export function syncReduxAndRouter(history, store, selectRouterState = SELECT_ST
   const unsubscribeHistory = history.listen(location => {
     const route = {
       path: createPath(location),
-      state: location.state
+      state: location.state === null ? undefined : location.state
```

Comment:
@justingreenberg If it isn't too much work changing the tests, maybe you could take a look when you've got the opportunity? No hurry — remember to relax and enjoy xmas :)